core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Authenticator.java

```diff
@@ -78,9 +78,12 @@ protected Optional<Identity> createIdentity(String token)
         if (claims.isEmpty()) {
```

Comment:
Can you describe also the bug which you are fixing. I mean to explain the behavior that user can experience that is now improved. Please capture is as a release notes and add it to the PR description.